### ISSUE TITLE
Refresh page in browser if already open.

### ISSUE THREAD

**Issue by JohnLukeBentley** [skandasoft/open-in-browsers#18]
When in atom and using open-in-browsers a file will open in a new tab (for Chrome, FF, Edge) or a new window (for IE 11).

I suggest this behaviour only occur if the page is not already loaded in the browser. If the page is already loaded in the browser then, so is my suggestion, make that browser and tab activate, and refresh the page.

Edit: Presumably this essentially depends on whether each browser supports this from the command line.

**Comment by skandasoft** [comment]
you are right. If it is possible from command prompt to open the same ..let me know..

**Comment by JohnLukeBentley** [comment]
I'll do the research and let you know.

**Comment by skandasoft** [comment]
Works for Chrome
